cgyle/proxy.py

```diff
@@ -164,16 +164,16 @@ def update_cache(
                     Path(os.path.dirname(log_name)).mkdir(
                         parents=True, exist_ok=True
                     )
+                    call_args = [
+                            'skopeo', 'copy', '--retry-times', '5'
+                    ]
                     if arch == 'all':
-                        call_args = [
-                            'skopeo', 'copy', '--all',
-                            f'--src-tls-verify={format(tls_verify).lower()}'
-                        ]
+                        call_args.append('--all')
                     else:
-                        call_args = [
-                            'skopeo', '--override-arch', arch, 'copy',
-                            f'--src-tls-verify={format(tls_verify).lower()}'
-                        ]
+                        call_args += ['--override-arch', arch]
+                    call_args.append(
+                        f'--src-tls-verify={format(tls_verify).lower()}'
+                    )
```

Comment:
I understand your reasons for the structure change but this breaks the skopeo call. ```--override-arch``` is a global option and your change now moves it as an option to the copy command. As I'm on the unit tests anyway I'm going to fix this too

Reply:
fixed